app/javascript/src/javascripts/blacklists.js

```diff
@@ -192,8 +117,8 @@ Blacklist.postHide = function (post) {
 
 Blacklist.postShow = function (post) {
   const $post = $(post);
-  Blacklist.postRestoreSrc(post);
-  $post.addClass("blacklisted").removeClass("blacklisted-active").removeClass("blacklisted-active-visible");
```

Comment:
There didn't seem to be any reason for having seperate classes when an item can be blacklisted and when it actually is, so I removed them.